task.css

```diff
@@ -0,0 +1,83 @@
+
+.container {
+    font-family:'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size: small;
+    display: flex;
+    flex-direction: row;
+    padding:0px 0px 0px 0px;
+    }
+ h1 {
+    color:rgb(204, 11, 108);
+    text-align: center;
+  }
+   h2,p {
+    color:black;
+    text-align: center;
+  }
+
+  .container .first {
+    text-align: center;
+    position: relative;
+    left:15%;
+  }
+  .container .first  .wrap{
+
+   width:350px;
+   background: white;
+   padding:5px;
+  
+
+  }
+  .wrap .in{
+  height: 45px;
+    width: 100%;
+    position: relative;
+ }
+ .wrap .in input{
+  padding-top: 10px;
+    height: 100%;
+    width: 100%;
+    border: none;
+    border-bottom: 3px solid silver;
+    box-shadow: 0px 0px 10px  rgb(0,0,0,0.1);
+
+ }
+ .wrap .in label{
+     padding-top: 5px ;
+  
+     font-size: small;
+     position: absolute;
+    }
+
+#rem {
+    height:12px ;
+    width: 15px;
+    position: relative;
+   right:40px;
+
+ }
+ #for-pass {
+    height:12px ;
+    width: 15px;
+    position: relative;
+   left:45px;
+
+ }
+ #con  {
+    height: 40px;
+     width:350px ;
+     color:whitesmoke ;
+     background-color: rgb(22, 129, 236);
+    border-color:rgb(22, 129, 236);
+ }
+  .container .second {
+    position: relative;
+    left: 30%;
+    
+} 
+.container .second >img{
+  
+    width: 710px ;

```

Comment:
you are cutting the photo to a fixed width/height.
and this is shrinking it.
![image](https://user-images.githubusercontent.com/10697961/155884945-86707b9c-790e-4393-a3e3-6c9ba611b12f.png)